transformation-scripts/semopenalex-domains.py

```diff
@@ -0,0 +1,306 @@
+        with gzip.open(filename, 'r') as f:
+
+            for line in f:
+                try:
+                    json_data = json.loads(line.decode('utf-8'))
+
+                    #domain-ID
+                    domain_id = json_data['id'].replace("https://openalex.org/domains/", "")
+                    domain_uri = URIRef(soa_namespace_domains+domain_id)
+                    domain_graph.add((domain_uri,RDF.type, soa_class_domain))
+                    domain_graph.add((domain_uri,RDF.type, SKOS.Concept))
+
+                    #topic Scheme
+                    domain_graph.add((domain_uri,SKOS.inScheme,topic_scheme_uri))
+
+                    # display_name
+                    domain_display_name = json_data['display_name']
+                    if not domain_display_name is None:
+                        domain_display_name = clean(domain_display_name)
+                        domain_graph.add((domain_uri, SKOS.prefLabel, Literal(domain_display_name, datatype=XSD.string)))
+
+                    # display_name_alternatives
+                    domain_display_name_alternatives = json_data['display_name_alternatives']
+                    if not domain_display_name_alternatives is None:
+                        domain_display_name_alternatives = clean(domain_display_name_alternatives)
+                        domain_graph.add((domain_uri, SKOS.altLabel, Literal(domain_display_name_alternatives, datatype=XSD.string)))
+
+                    # description
+                    domain_description = json_data['description']
+                    if not domain_description is None:
+                        domain_description = clean(domain_description)
+                        domain_graph.add((domain_uri, SKOS.note, Literal(domain_description, datatype=XSD.string)))
+
+                    # siblings
+                    domain_siblings = json_data['siblings']
+                    if not domain_siblings is None:
+                        for sibling in domain_siblings:
+                            sibling_id = sibling['id'].replace("https://openalex.org/domains/", "")
+                            sibling_uri = URIRef(soa_namespace_domains+sibling_id)
+                            domain_graph.add((domain_uri, SKOS.related, sibling_uri))
+
+                    # ids (wikidata, wikipedia)
+                    domain_wikidata = json_data.get('ids').get('wikidata')
+                    if not domain_wikidata is None:
+                        domain_wikidata = clean_url(domain_wikidata)
+                        domain_graph.add((domain_uri,OWL.sameAs, URIRef(domain_wikidata)))
+
+                    domain_wikipedia = json_data.get('ids').get('wikipedia')
+                    if not domain_wikipedia is None:
+                        domain_wikipedia = clean_url(domain_wikipedia)
+                        domain_graph.add((domain_uri,RDFS.seeAlso, Literal(domain_wikipedia, datatype=XSD.anyURI)))
```

Comment:
wikipedia is not a literal.
```suggestion
                        domain_graph.add((domain_uri,RDFS.seeAlso, URIRef(domain_wikipedia)))
```

Reply:
wikipedia = Literal (datatype=XSD.anyURI)
wikidata = URIRef()

or?

Reply:
wikidata = URIRef()

Reply:
let's use `owl.sameAs` as predicate. 
```
domain_graph.add((domain_uri, OWL.sameAs, URIRef(domain_wikipedia)))
```